### ISSUE TITLE
Opening PEP with SessionId

### ISSUE THREAD

**Issue by jwoosnam** [Psychoanalytic-Electronic-Publishing/PEP-Web-User-Interface#606]
When calling the client with a session ID it appears that sometimes the session ID is not being passed to PaDS and therefore PaDS is allocating a new session ID.
This is causing a significant problem for people logging on using Federated/OpenAthens.  For me the link below, for a London City University OpenAthens session, sometimes works and sometimes doesn't work.
[https://pep-web.org/?sessionld=1c097707-9e89-45ab-9fa2-5fc8c0c7f24c ](url)

I would be happy to jump on a call with someone to demonstrate.
@nrshapiro @rickardnyman

**Comment by nrshapiro** [comment]
@adistasio @bakerac4 - Can you meet online with James to diagnose/discuss?

**Comment by bakerac4** [comment]
@nrshapiro @jwoosnam after looking into this for a little, it seems that your sending `sessionId` with a lowercase L, instead of an uppercase `I`. If you send the correct query param it should work.

**Comment by bakerac4** [comment]
<img width="517" alt="Screen Shot 2022-01-24 at 10 38 10 AM" src="https://user-images.githubusercontent.com/9952936/150814167-259ed0a7-930d-4093-bfeb-d229b92ba50c.png">
Thats what you sent from above ^

This is me fixing it 
<img width="510" alt="Screen Shot 2022-01-24 at 10 38 20 AM" src="https://user-images.githubusercontent.com/9952936/150814198-9275789b-b420-436c-9887-97fdf6ba5eac.png">

**Comment by jwoosnam** [comment]
Thanks @bakerac4. I can see me example above is wrong and I am sorry to have made you investigate this.  I have checked the PaDS source and I can find no instant of us using lower case l.  I think this may have just been in my example. 

Users were having a problem similar to the one I described, but obviously not with the wrong query string. I will investigate further and try and come back with more information.

For now I have downgraded the priority.
